core-web/libs/sdk/react/src/lib/deprecated/components/DotcmsLayout/DotcmsLayout.tsx

```diff
@@ -36,6 +36,7 @@ export type DotcmsPageProps = {
  * @category Components
  * @param {DotcmsPageProps} props - The properties for the DotCMS page.
  * @returns {JSX.Element} - A JSX element that represents the layout for a DotCMS page.
+ * @deprecated Use `DotCMSLayoutBody` instead. More info at https://github.com/dotCMS/core/blob/main/core-web/libs/sdk/react/src/lib/next/components/DotCMSLayoutBody/DotCMSLayoutBody.tsx
```

Comment:
Use [this](https://jsdoc.app/tags-inline-link) for the links

Reply:
Done, bud!